### DISCUSSION TITLE
[BUG] Repo's root link is generated with "blob" not "tree"

### DISCUSSION THREAD

**Discussion by anesbelo** [community/community#13122] [Other Feature Feedback, Questions, & Ideas]
Hello,

There's a bug with relative links that point to the root directory of a GitHub repository.

**Where**

- sub/directory/README.md file

**What**

- Back to \[root\]\(/\)

**Problem**

- Redirection to "https:\/\/github.com\/\<username\>/\<repo-name\>/**blob**/\<branch-name\>"

**Expected**

- Redirection to "https:\/\/github.com\/\<username\>/\<repo-name\>/**tree**/\<branch-name\>"

Thanks in advance!

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
